src/styles.scss

```diff
@@ -650,6 +650,62 @@ ul.our-sponsors li img {
   margin-right: 10px;
 }
 
+.tos-banner {
+  flex-shrink: 0;
+  flex-grow: 0;
+  padding: 3px 15px;
+  color: white;
+}
+
+@media (max-width: 992px) {
+  .tos-banner {
+    height: 75px;
+    line-height: 250%;
+  }
+
+  .push-tos {
+    height: 75px;
+  }
+}
+
+@media (min-width: 992px) {
+  .tos-banner {
+    height: 50px;
+    line-height: 50px;
+  }
+
+  .push-tos {
+    height: 50px;
+  }
+
+  .break-tos {
+    display: none;
+  }
+}
+
+.tos-banner a {
+  color: white;
+  text-decoration: underline;
+}
+
+.sticky {
+  background-color: #3b4045;
+  width: 100%;
+  position: fixed;
+  top: 0;
+  overflow: hidden;
+  z-index: 10;
```

Comment:
Kind of chose an arbitrary number, but it's so it can be in front of div-container and mat cards. But works with just 3 apparently, so changed it to that.